lib/screens/homepage/home_page.dart

```diff
@@ -15,132 +17,127 @@ class Homepage extends StatefulWidget {
+                    color: const Color(0xff49B6D5),
+                    secondaryColor: const Color(0xff2C6E80),
+                  ),
+                  HomepageCardSecondary(
+                    title: 'HOUSEHOLDS',
+                    value:
+                        context.read<UIProvider>().getDashData['hh_holds'] ?? 0,
+                    icon: FontAwesomeIcons.house,
+                    color: const Color(0xffFE5C57),
+                    secondaryColor: const Color(0xff9A3734),
+                  ),
+                ],
+              ),
+              const SizedBox(
+                height: 80,
+              )
+            ],
           ),
-
-          HomepageCard(
-            data: {
-              'title': 'CAREGIVERS/GUARDIANS',
-              'value': context
-                      .read<UIProvider>()
-                      .getDashData['caregivers']
-                      .toString() ??
-                  "null",
-              'icon': FontAwesomeIcons.peopleGroup,
-              'color': const Color(0xff348FE2),
-              's_color': const Color(0xff1F5788),
-            },
-          ),
-
-          HomepageCard(
-            data: {
-              'title': 'WORKFORCE MEMBERS',
-              'value': context
-                      .read<UIProvider>()
-                      .getDashData['workforce_members']
-                      .toString() ??
-                  "null",
-              'icon': Icons.people,
-              'color': const Color(0xff727DB6),
-              's_color': const Color(0xff454A6D),
-            },
-          ),
-          HomepageCard(
-            data: {
-              'title': 'ORG UNITS/CBOs',
-              'value': context
-                      .read<UIProvider>()
-                      .getDashData['org_units']
-                      .toString() ??
-                  "null",
-              'icon': FontAwesomeIcons.landmark,
-              'color': const Color(0xff49B6D5),
-              's_color': const Color(0xff2C6E80),
-            },
-          ),
-          HomepageCard(data: {
-            'title': 'HOUSEHOLDS',
-            'value':
-                context.read<UIProvider>().getDashData['hh_holds'].toString() ??
-                    "null",
-            'icon': FontAwesomeIcons.house,
-            'color': const Color(0xffFE5C57),
-            's_color': const Color(0xff9A3734),
-          }),
-
-          // const SizedBox(height: 10),
-          // ...List.generate(
-          //   context.read<UIProvider>().homeCardsTitles.length,
-          //   (index) => HomepageCard(
-          //       data: context.read<UIProvider>().homeCardsTitles[index]
-          //       // homeCardsTitles[index],
-          //       ),
-          // ),
-          // ...List.generate(
-          //     5,
-          //     (index) => GraphWidget(
-          //           title: graphTitles[index],
-          //         )),
-        ],
-      ),
+        ),
+        Positioned(
```

Comment:
Do this instead:
`Positioned(
          bottom: 20,
          right: 20,
          left: 20,
          child: 
          Row(
          mainAxisAlignment:mainAxisAlignment.end
          children:[
          CustomButton(onTap: () {}, text: "New OVC"),
          ])
        )`